example/index.ts

```diff
@@ -44,4 +48,18 @@ function runLazyView() {
     });
 } 
 
-runLazyView();
\ No newline at end of file
+runLazyView();
+
+ */
+function runMultiple() { 
+    const configs = 
+        [ { viewFunction: "MyModule.view", model: model
+            , decoder: "MyModule.decodeModel", output: "multiple1.html" } 
+            , { viewFunction: "MyModule.lazyView", model: model
+            , decoder: "MyModule.decodeModel", output: "multiple2.html" } 
```

Comment:
💭 is this the preferred API for you? do you think there could be something better?

Reply:
I'm not sure, I agree it doesn't feel very nice.

It depends on what one wants to do with the files, I guess. For me, I just want to save them to disk, so the above makes sense. It's possible to return a `Promise<[string]>`, then the user 
has to re-associate the generated html and the file name:

```typescript
function runMultiple() {
    const configs =
        [ { viewFunction: "MyModule.view", model, decoder: "MyModule.decodeModel", output: "multiple1.html" }
        , { viewFunction: "MyModule.lazyView", model, decoder: "MyModule.decodeModel", output: "multiple2.html" }
        ];

    elmStaticHtml.grouped(process.cwd(), "MyModule", configs)
        .then((generatedHtmls) => {
            generatedHtmls
                .map((generatedHtml, i) => fs.writeFileSync(configs[i].output, generatedHtml));
        });
}
```